assets/js/util/index.js

```diff
@@ -85,6 +85,11 @@ export const getTimeInSeconds = ( period ) => {
  * @return {(number|null)} The percent change. Null if the input or output is invalid.
  */
 export function calculateChange( previous, current ) {
+	// Prevent null result when both values are legitimately 0
+	if ( [ '0', 0 ].includes( previous ) && [ '0', 0 ].includes( current ) ) {
+		return 0;
+	}
+
 	// Prevent divide by zero errors.
 	if ( '0' === previous || 0 === previous || Number.isNaN( previous ) ) {
```

Comment:
I'd suggest adding an `isZero()` function to make things a bit more readable here...

We should also add a full stop to the new comment :)

```suggestion
	const isZero = ( value ) => value === '0' || value === 0;

	// Prevent null result when both values are legitimately 0.
	if ( isZero( previous ) && isZero( current ) ) {
		return 0;
	}

	// Prevent divide by zero errors.
	if ( isZero( previous ) || Number.isNaN( previous ) ) {
```